tripal/config/install/views.view.tripal_content_type_listing.yml

```
@@ -383,7 +384,18 @@ display:
       cache:
         type: none
         options: {  }
-      empty: {  }
+      empty:
+        area_text_custom:
+          id: area_text_custom
+          table: views
+          field: area_text_custom
+          relationship: none
+          group_type: group
+          admin_label: ''
+          plugin_id: text_custom
+          empty: true
+          content: '<p>There are no tripal content entities yet.</p>'
```

This is actually shown both when there are no Tripal entities yet AND when the filter criteria entered do not match 🙈 As such, I suggest a more generic message like the following. This matches what Drupal does.

```suggestion
          content: '<p>No Tripal content available.</p>'
```